sysboundary/sysboundary.cpp

```diff
@@ -409,6 +409,7 @@ bool SysBoundary::classifyCells(dccrg::Dccrg<spatial_cell::SpatialCell,dccrg::Ca
    // sysBoundaryLayer communicated)
    SpatialCell::set_mpi_transfer_type(Transfer::CELL_SYSBOUNDARYFLAG);
    mpiGrid.update_copies_of_remote_neighbors(SYSBOUNDARIES_NEIGHBORHOOD_ID);
+   technicalGrid.updateGhostCells();
```

Comment:
My only comment is that I would move this to around line 477, so that the technical grid layer computation is in one place. Now the dccrg layer computation is inbetween

Reply:
Other than that it is fine :)

Reply:
Like so?